test/integration/users.test.js

```diff
@@ -193,6 +204,123 @@ describe("Users", function () {
           return done();
         });
     });
+
+    it("Should return next and prev links", function (done) {
+      chai
+        .request(app)
+        .get("/users?size=2")
+        .set("cookie", `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("Users returned successfully!");
+          expect(res.body).to.have.property("links");
+          expect(res.body.links).to.have.property("next");
+          expect(res.body.links).to.have.property("prev");
+
+          return done();
+        });
+    });
+
+    it("Should return 400 when both next and prev passed as query param", function (done) {
+      chai
+        .request(app)
+        .get(`/users?next=${userId}&prev=${userId}&size=2`)
+        .set("cookie", `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+
+          expect(res).to.have.status(400);
+
+          return done();
+        });
+    });
+
+    it("Should return 400 when both next and page passed as query param", function (done) {
+      chai
+        .request(app)
+        .get(`/users?next=${userId}&page=1&size=2`)
+        .set("cookie", `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+
+          expect(res).to.have.status(400);
+
+          return done();
+        });
+    });
+
+    it("Should return 400 when both page and prev passed as query param", function (done) {
+      chai
+        .request(app)
+        .get(`/users?page=1&prev=${userId}&size=2`)
+        .set("cookie", `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+
+          expect(res).to.have.status(400);
```

Comment:
Nit : we can also check the error message.

Reply:
> Can we have a couple of more tests please where we check `next` and `prev` query params. There are currently no tests to verify the `next` and `prev` query parameters.

Yes adding a test for that thanks for mentioning

Reply:
done